examples/00_misc/03_precipitation.py

```diff
@@ -20,12 +20,12 @@
 # fix the seed for reproducibility
```

Comment:
Sneaking changes to my code into a totally unrelated PR, ey?! ;-) yeah yeah, you just blackened...

Reply:
Sorry!